lib/active_encode/engine_adapters/ffmpeg_adapter.rb

```diff
@@ -206,7 +212,11 @@ def ffmpeg_command(input_url, id, opts)
           file_name = "outputs/#{sanitized_filename}-#{output[:label]}.#{output[:extension]}"
           " #{output[:ffmpeg_opt]} #{working_path(file_name, id)}"
         end.join(" ")
-        "#{FFMPEG_PATH} -y -loglevel error -progress #{working_path('progress', id)} -i \"#{input_url}\" #{output_opt}"
+        header_opt = opts[:headers]&.map do |k, v|
+          "-headers '#{k}: #{v}'"
```

Comment:
Do multiple `-headers` flags work correctly?  This old blog post says to combine headers into one gnarly string.
https://www.jokecamp.com/blog/passing-http-headers-to-ffmpeg/

Reply:
That explains why ffmpeg complained about missing CRLF